server/views/pages/addNote.njk

```diff
@@ -0,0 +1,75 @@
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Add a note" %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {% if errors %}
+                {{ govukErrorSummary({
+                    titleText: "There is a problem",
+                    errorList: errors.list,
+                    attributes: {
+                        'data-qa': 'error-list'
+                    }
+                }) }}
+            {% endif %}
+            <h1 class="govuk-heading-l">
+                Add a note to the recall
+            </h1>
+        </div>
+    </div>
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds-from-desktop">
+            <form novalidate action="{{ makeUrl('add-note', urlInfo, csrfToken) }}"
+                  method="post" enctype="multipart/form-data">
+                {{ govukInput({
+                    name: "subject",
+                    id: "subject",
+                    label: {
+                        text: "Subject",
+                        classes: "govuk-label--m"
+                    },
+                    value: formValues.subject,
+                    errorMessage: errorMessage(errors.subject)
+                }) }}
+                {{ govukTextarea({
+                    name: "details",
+                    id: "details",
+                    label: {
+                        text: "Details",
+                        classes: "govuk-label--m"
+                    },
+                    value: formValues.details,
+                    errorMessage: errorMessage(errors.details)
+                }) }}
+                <h2 class="govuk-heading-m govuk-!-margin-bottom-3">Upload document (optional)</h2>
+                {% include '../partials/uploadedDocumentLink.njk' %}
```

Comment:
I don't think this is needed, they're not going to return to this page to edit, it would just add a new note